manifests/agent.pp

```diff
@@ -58,6 +58,9 @@
 # @param namespace
 #   The agent namespace
 #   Passing `namespace` as part of `config_hash` takes precedence
+# @param redact
```

Comment:
Add something like, The default values from hiera Sensu's documentation and a link to it.

Reply:
Added link, can't link directly to `redact` so just linked to section above it.